1.0/profile.ttl

```diff
@@ -0,0 +1,64 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX prof: <http://www.w3.org/ns/dx/prof/>
+PREFIX role: <http://www.w3.org/ns/dx/prof/role/>
+PREFIX sdo: <https://schema.org/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+
+<http://www.opengis.net/def/geosparql>
+    a prof:Profile ;
+    owl:versionIRI <http://www.opengis.net/def/geosparql/1.0> ;   
+    dcterms:created "2020-11-20"^^xsd:date ;
```

Comment:
I'm a little unconformable with the fact that the actual date of the GeoSPARQL 1.0 is not in its own property. 
I can see that the profile resource was only created now, but the things that it describes date from 2012.

Reply:
> "the actual date of the GeoSPARQL 1.0"

You are seeing the date of the _GeoSPARQL 1.0 Profile_, not the date of the thing most commonly referred to as "GeoSPARQL" which would be the specification (HTML doc) and/or the ontology, which have the 2012 date.

One of the things these profile declarations will do is to sort out the ambiguity about what is refereed to by "GeoSPARQL".

Reply:
Yes, I said that. But I raised an important related issue.

Reply:
So do you propose a change to this PR?

Reply:
(these profile declarations are expected to change or be edited throughout this SWG so you could always propose a change or even just raise an Issue here and we will deal with it after the merger of this PR)

Reply:
It feels to me like the profile (understood as a suite of artefacts) was implicitly or conceptually created in 2012. The fact that this particular formalization of the description was only created now is of little interest.

Reply:
That is what I originally indicated with a profile date of 2012 - same as the spec doc/ontology. Feedback from Franz was to have the Profile use a separate date (now).

I don't think we have a strong answer for this point. Perhaps we can lean on `dcterms:issued` and therefore have something like:

- `dcterms:created` - implicit creation date of the profile, ~2012
- `dcterms:issued` - unveiling of profile, now
- `dcterms:modified` - latest edit date of profile

The first line of the profile description - "This is a 'profile declaration' for the GeoSPARQL 1.0 specification (standard) as originally published in 2012." - I think will support this.

Would you agree to this change?

Reply:
`created` vs `issued` might be a pragmatic and usable pattern. I'm not quite as confident about the semantics of `modified` though. But would like to have some other opinions. 

Note that my original comment was _not_ a critique of the use of the dcterms predicates as shown. It _was_ that the date '2012' was not available discretely (yes, I noticed it embedded within a string literal, but that's not really good enough, eh?). This meta-level stuff is always a bit tricky.

Reply:
OK, I've added in the creation date as suggested above for now.

Reply:
Did the concept of a profile declaration really exist in 2012? We shouldn't practice revisionism.
Furthermore, the definition of `dcterms:created` is "Date of creation of the resource". With  `dcterms:created` in this case being a property of the profile web resource, it seems like we are saying the profile as a web resource was created in 2012.

Would it help to assign dates to the individual resources making up the profile declaration? E.g. indicate that the GeoSPARQL documentation and ontology where published in 2012? That should clarify matters regardless of the dates assigned to the profile. 

In the line  "This is a 'profile declaration' for the GeoSPARQL 1.0 specification (standard) as originally published in 2012." I read 2012 as applying to the specification, not to the profile declaration.

Reply:
On review, I think I'm not in favour of using created date of 2012 here since, as @FransKnibbe says, there was no notion of a profile back then. So I'm reverting my creation date of the _profile_ to now and the resources within the profile will have to stand on their own two feed with their 2012 creation dates.

Reply:
...and with this I'm merging since I have two approvals here and this is a small issue that, if necessary, can be picked up again in subsequent PRs.